octue/resources/datafile.py

```diff
@@ -186,7 +185,7 @@ def from_cloud(cls, project_name, bucket_name, datafile_path, allow_overwrite=Fa
             timestamp=kwargs.get("timestamp", metadata.get("customTime")),
             id=kwargs.get("id", custom_metadata.get("id", ID_DEFAULT)),
             path=storage.path.generate_gs_path(bucket_name, datafile_path),
-            hash_value=kwargs.get("hash_value", custom_metadata.get("hash_value", metadata.get("crc32c", None))),
+            hash_value=metadata.get("crc32c", None),
```

Comment:
This is intentionally default `None`, rather than `EMPTY_STRING_HASH_VALUE`?

Reply:
It was getting set to `EMPTY_STRING_HASH_VALUE` deeper in `Datafile._calculate_hash`, but I've now realised I can refactor it so it doesn't have to go deeper at all by changing the default above to `EMPTY_STRING_HASH_VALUE` 👌